src/Tasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -495,6 +491,7 @@ public static PermissionSet XmlToPermissionSet(XmlElement element)
         /// <param name="certThumbprint">Hexadecimal string that contains the SHA-1 hash of the certificate.</param>
         /// <param name="timestampUrl">URL that specifies an address of a time stamping server.</param>
         /// <param name="path">Path of the file to sign with the certificate.</param>
+        [SupportedOSPlatform("windows")]
```

Comment:
Should this just be applied to the class?

Reply:
I don't _think_ so in this case. There are lots of internal static methods here that are valid cross-OS.